.changelog/2134.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:new-resource
+cloudflare_workers_queue
```

Comment:
shouldn't this be just `cloudflare_queue`?

Reply:
A bit of a tough call. The product is indeed named "Cloudflare Queues". But "Queues" lives under the "Workers" tab in Dash and is under the "workers" section of the Cloudflare V4 API (accounts/<id>/workers/queues/<name>).

Your suggestion is probably for the best though, I'll go ahead and switch it to drop the workers prefix.

Reply:
Done